docs/network-flow-visibility.md

```diff
@@ -64,7 +66,7 @@ manifest, in the following way:
 
 ### Theia Installation
 
-To enable both Grafana Flow Collector and
+To enable both [Grafana Flow Collector](#grafana-flow-collector) and
 [NetworkPolicy Recommendation](networkpolicy-recommendation.md), please install
 Theia and Flow Aggregator by runnning the following commands:
```

Comment:
Not introduced by this PR, could you change the order "Theia and Flow Aggregator" to "Flow Aggregator and Theia". I forgot to change it when changing the installation order. And the same for Line 77. Thanks!

Reply:
Sure. Changed.